src/modules/SpaceCalculation.ycp

```diff
@@ -642,8 +642,18 @@
 				// join the options
 				string mount_options_str = mergestring(mount_options, ",");
 
+				string device = "";
+				if (part["crypt_device"]:false)
```

Comment:
Use `haskey()` here:  `if (haskey(part, "crypt_device"))`, the default value is used also for type information, it this case the value is not boolean.

See http://doc.opensuse.org/projects/YaST/SLES11/tdg/haskey.html